admin-api-server/src/analytics/service/analytics.service.ts

```diff
@@ -117,6 +119,39 @@ export class AnalyticsService {
     });
   }
 
+  async getUsers(params: PaginationParams) {
+    const qryRes = await this.storeRepl.query(
+      `SELECT
+        id,
+        address,
+        email,
+        consent,
+        created_at
+        FROM marketing 
+        ORDER BY "${params.orderBy}" ${params.orderDirection}
+        OFFSET ${params.pageOffset}
+        LIMIT ${params.pageSize}`,
+    );
+
+    if (qryRes.rowCount === 0) {
+      return { data: [], count: 0 };
+    }
+
+    return {
+      data: qryRes.rows.map(
+        (row: any) =>
+          <MarketingEntity>{
+            id: row['id'],
+            address: row['address'],
+            email: row['email'],
+            consent: row['consent'] === true ? 'Yes' : 'No',
+            createdAt: row['created_at'],
+          },
+      ),
+      count: Number(qryRes['rowCount']),
```

Comment:
I think this should simply say `count: qryRes.rowCount`? No need to cast to Number I think, it should already be a number (correct me if wrong).